### PR TITLE
Extract DB abstraction

### PR DESCRIPTION
**Pull request by forki**

### CODE REVIEW

File: src/Server/Database.fs [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+open Suave.Operators
+open Suave.RequestErrors
+open Microsoft.Azure.WebJobs
+open ServerCode.Storage.AzureTable
+open ServerCode
+
+[<RequireQualifiedAccess>]
+type DatabaseType = 
+    | FileSystem 
+    | Azure of connectionString : AzureConnection
```
Comment (cartermp) [inline src/Server/Database.fs]: Should this be `AzureStorage`?

Comment (isaacabraham) [reply src/Server/Database.fs]: For completeness, probably. It's not particularly confusing at the moment as there's no e.g. DocumentDB story just yet :-)

**Comment by forki** [comment]
@isaac_abraham please rename it. I like the proposal

Am 11.10.2017 18:31 schrieb "Isaac Abraham" <notifications@github.com>:

**Comment by isaacabraham** [comment]
Already done in master.

**Pull request merged**